features/admin.extensions.v1/configs/models/documentation.ts

```diff
@@ -616,6 +623,7 @@ export interface DocumentationLinksExtensionInterface {
          * Documentation links for account recovery elements.
          */
         accountRecovery: AccountRecoveryDocumentationLinksInterface;
+        administrators: AdministratorsDocumentationLinksInterface;
```

Comment:
Let's add a doc comment here, as other fields has that.

Reply:
Fixed with [4735aca](https://github.com/wso2/identity-apps/pull/7141/commits/4735aca33a59a39f2d466a38711df8632b37998e)